src/Sensor.cc

```diff
@@ -41,7 +42,7 @@ const std::vector<std::string> sensorTypeStrs =
   "contact",
   "depth_camera",
   "force_torque",
-  "gps",
+  "satnav",
```

Comment:
Let's keep `gps` for backwards compatibility, the same way as in the enum

Reply:
https://github.com/osrf/sdformat/pull/453/commits/b128c9cc8c5aad02e8db480bd25626ae8769c545